coretex/entities/project/project.py

```diff
@@ -110,3 +124,10 @@ def addTask(self, name: str, description: Optional[str]) -> bool:
 
         self.tasks.append(task)
         return True
+
+    def changeVisibility(self, visibility: ProjectVisibility) -> NetworkResponse:
```

Comment:
`updateVisibility`. Function is missing documentation. Raise an exception if the request has failed, do not directly return `NetworkResponse`.